mods/default/furnace.lua

```diff
@@ -114,6 +114,15 @@ local function swap_node(pos, name)
 	minetest.swap_node(pos, node)
 end
 
+local function add_item(inv, pos, item)
```

Comment:
```suggestion
local function add_item_or_drop(inv, pos, item)
```
Makes it more clear what the function does.